static/app/utils/analytics/samplingAnalyticsEvents.tsx

```diff
@@ -8,6 +8,9 @@ type Rule = {
 };
 
 export type SamplingEventParameters = {
+  'sampling.sdk.client.rate.change.alert': {
+    project_id: string;
+  };
```

Comment:
These changes look unrelated. Probably leftovers from a different branch?

Reply:
oh no... let me remove it... thanks